src/components/views/elements/MemberEventListSummary.js

```diff
@@ -159,6 +165,20 @@ module.exports = React.createClass({
         return res;
     },
 
+    /**
+     * Transform an array of transitions into an array of transitions and how many times
+     * they are repeated consecutively.
+     *
+     * An array of 123 "joined_and_left" transitions, would result in:
+     * ```
+     * [{
+     *   transitionType: "joined_and_left"
+     *   repeats: 123
+     * }, ... ]
+     * ```
+     * @param {string[]} transitions the array of transitions to transform.
+     * @returns {object[]} an array of truncated transitions.
+     */
     _getTruncatedTransitions: function(transitions) {
```

Comment:
I think this is a poor name for this. It implies that you're dropping transitions off the list, which isn't the case.

How about something like `_coalesceRepeatedTransitions` or sth?